autogen_stubs.sh

```diff
@@ -22,12 +22,38 @@ fixup() {
   grep FIXME_STUB $1 || true
 }
 
+generate_opencl_error_codes(){
+
+  OPENCL_HEADERS_SRC=/tmp/opencl-headers
+  OPENCL_COMMIT_HASH=c860bb551eeef9a47d56286a70cea903db3d6ed2
+  if [ ! -d "$OPENCL_HEADERS_SRC" ]; then
+    git clone https://github.com/KhronosGroup/OpenCL-Headers.git $OPENCL_HEADERS_SRC
+    pushd .
+    cd $OPENCL_HEADERS_SRC
+    git reset --hard $OPENCL_COMMIT_HASH
+    popd
+  fi
+  
+  file=$OPENCL_HEADERS_SRC/opencl_headers_coalesce.h
+  cat $OPENCL_HEADERS_SRC/CL/*.h > $file # Form a single file from all headers, to extract all errors present
```

Comment:
in `opencl` we take headers from `/usr/include/CL`, so let's not change this *partially* in this pr, can you take all needed headers from `/usr/include/CL/`?

Reply:
So, I think that's a distro compatibility problem, for example, I'm running openSuse and /usr/include doesn't even have a CL folder, of course I can fix it and put it myself, but other developers might run on the same issue.

If that's acceptable , i can comment it (for reference) and set the OPENCL_HEADERS_SRC=/usr/include/CL .

Should i keep it as is or force the /usr/include/CL usage?